src/renderer/components/SpinnerWrapper.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="component-wrapper">
+    <slot></slot>
+    <div v-if="messageReceiving" class="spinner-container">
+      <div :class="{ 
+        'spinner': !size, 
+        'spinner-sm': size === 'small' 
```

Comment:
Just a style thing, but you could always include `spinner` here and just do this via css. 

```vue
    <div :class="['spinner', {'spinner-sm': size === 'small'}]">
```

```css
    .spinner {
        &:not(.spinner-sm) {
            @include loader06($size: 36px, $color: $purple, $align: middle);
        }
        &.spinner-sm {
            @include loader06($size: 36px, $color: $purple, $align: middle);
        }
    }
```

Reply:
Updated.